src/Animals.java

```diff
@@ -0,0 +1,46 @@
+public class Animals {
+
+    private String name;
+    private static int animalsCount;
+    private static int catsCount;
+    private static int dogsCount;
+
+    public Animals(String name) {
+        this.name = name;
+        animalsCount++;
+
+        if (this instanceof Cat) {
+            catsCount++;
+        } else if (this instanceof Dog) {
+            dogsCount++;
+        }
+    }
+
+    public static int getAnimalsCount() {
+        return animalsCount;
+    }
+
+    public static int getCatsCount() {
+        return catsCount;
+    }
+
+    public static int getDogsCount() {
```

Comment:
move this method to the `Dog` class where the dogs counter should be implemented